recipe/meta.yaml

```diff
@@ -27,12 +27,13 @@ requirements:
     - python
     - pip
     - rdkit {{ rdkit_version }}
-    - boost >=1.67
-    - boost-cpp >=1.67
+    - libboost >=1.67
+    - libboost-headers >=1.67
+    - libboost-python-devel >=1.67
   run:
     - python
     - {{ pin_compatible('rdkit', max_pin='x.x') }}
-    - {{ pin_compatible('boost', max_pin='x.x') }}
+    - {{ pin_compatible('libboost', max_pin='x.x') }}
```

Comment:
Thanks for fixing the recipe. However, this is not the correct way to use the boost libraries. I'm taking the opportunity to fix this in #16